components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -295,6 +297,10 @@ public boolean insertAccessToken(String accessToken, String consumerKey,
                         .TOKEN_STATE_EXPIRED, UUID.randomUUID().toString(), userStoreDomain);
             }
             insertAccessToken(accessToken, consumerKey, newAccessTokenDO, connection, userStoreDomain);
+
+            if (isTokenCleanupFeatureEnabled && existingAccessTokenDO != null) {
+                oldTokenCleanupObject.cleanupTokenByTokenId(existingAccessTokenDO.getTokenId(), connection);
```

Comment:
Check the max length

Reply:
the line does not exceed max length here